traveller/modules/box__default/appadmin/view.py

```diff
@@ -99,6 +105,14 @@ def user_add():
             new_user.password = password
             new_user.bio = bio
 
+            # handle image upload
+            if 'image' in request.files:
+                image_file = request.files['image']
+                # replace symbols in email
+                filename = email.translate({ord(x): '_' for x in ['.', '@']})
+                image_path = images.save(image_file, 'profile', filename)
```

Comment:
What if the person uploads the same filename twice what will happen?

Reply:
The file gets overridden. This is acceptable here since a user typically has only one profile picture.

Reply:
Ok